src/containers/backpack.jsx

```diff
@@ -2,18 +2,24 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import bindAll from 'lodash.bindall';
 import BackpackComponent from '../components/backpack/backpack.jsx';
-import {getBackpackContents} from '../lib/backpack-api';
+import {getBackpackContents, saveBackpackObject, deleteBackpackObject, soundPayload, costumePayload} from '../lib/backpack-api';
```

Comment:
The length is 128 which is above 120 which is the maximum character count per line. You should maybe change the names to shorter versions like gBackPackContents, saveBackpackObj, delBackpackObject, sndPayload, and costumePayload can stay the same.

Reply:
It would reduce the length to 118.

Reply:
Or separate them onto different lines, like so:

```js
import {
  getBackpackContents,
  saveBackpackObject,
  deleteBackpackObject,
  soundPayload,
  costumePayload
} from '../lib/backpack-api'; 
```

This wouldn't take any refactoring like your suggestion (where you'd need to rename the functions in backpack-api *and* here), and the function names would stay as englishy as they are now, so probably a little more convenient.